docs/_sass/_content.scss

```diff
@@ -0,0 +1,41 @@
+.docs {
+  main {
+    hr {
+      margin: 2rem 0 1rem 0;
+    }
+
+    h1 {
+      font-size: 2rem;
```

Comment:
Do you know if I can modify the variables just for pages that have the 'docs' class added?